### PR TITLE
Updates the csv section from the doctoral skills and adds pandas

### PR DESCRIPTION
**Pull request by dpshelio**
The csv module section had been removed due to problems with the data
downloaded. This has now been fixed on our other course notes. The content has
been almost moved through, but leaving a section about how to read the data with
Numpy.
Additionally, the section has been expanded to include a very short introduction
to pandas.

### CODE REVIEW

File: ch01data/062csv.ipynb [hunk 1]
```diff
@@ -134,42 +148,274 @@
...
+    "### NumPy's CSV readers"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "The Python standard library has a `csv` module. However, it's less powerful than the CSV capabilities in `numpy`,\n",
-    "the main scientific python library for handling data. NumPy is destributed with Anaconda and Canopy, so we recommend you just use that."
+    "the main scientific python library for handling data. NumPy is destributed with Anaconda and Canopy, so we recommend use that when available."
```
Comment (matt-graham) [inline ch01data/062csv.ipynb:396]:
```suggestion
    "the main scientific python library for handling data. NumPy is distributed with Anaconda and Canopy, so we recommend use that when available."
```

File: ch01data/062csv.ipynb [hunk 2]
```diff
@@ -134,42 +148,274 @@
...
+   },
+   "source": [
+    "### NumPy's CSV readers"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "The Python standard library has a `csv` module. However, it's less powerful than the CSV capabilities in `numpy`,\n",
```
Comment (matt-graham) [inline ch01data/062csv.ipynb:395]:
```suggestion
    "The Python standard library `csv` module seen in the preceding section is less powerful than the CSV capabilities in NumPy,\n",
```

**Review by matt-graham — APPROVED** [review]
Suggested fix for small typo (carried over) and tried to make the flow from the section using `csv` module to NumPy CSV reading more natural, but otherwise looks good to me.

**Review by alessandrofelder — APPROVED** [review]
LGTM (have committed both @matt-graham 's suggestions)! 🙌

**Pull request merged**